src/System.Net.Requests/tests/HttpWebRequestTest.cs

```diff
@@ -1039,7 +1071,8 @@ public void Range_Add_Success(Uri remoteServer)
         {
             HttpWebRequest request = WebRequest.CreateHttp(remoteServer);
             request.UseDefaultCredentials = true;
-            await request.GetResponseAsync();
+            var response = await request.GetResponseAsync();
+            response.Dispose();
```

Comment:
This is the first change in this PR where I understand why it needed to be done.